ad/movement/trusts.md

```diff
@@ -503,6 +575,10 @@ When an ADCS is installed and configured in an Active Directory environment, a C
 
 {% embed url="https://www.sstic.org/media/SSTIC2014/SSTIC-actes/secrets_dauthentification_pisode_ii__kerberos_cont/SSTIC2014-Slides-secrets_dauthentification_pisode_ii__kerberos_contre-attaque-bordes_2.pdf" %}
 
+{% embed url="https://secureidentity.se/msds-shadowprincipal/" %}
+
+{% embed url="https://github.com/MicrosoftDocs/MIMDocs/blob/main/MIMDocs/pam/privileged-identity-management-for-active-directory-domain-services.experimental.md" %}
```

Comment:
Changed this to the direct documentation link, no need to go through the hassle of reading markdown

Reply:
Thank you.